modules/jdbc/src/main/java/org/apache/ignite/internal/jdbc2/JdbcResultSet.java

```diff
@@ -0,0 +1,1813 @@
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public Timestamp getTimestamp(int colIdx, Calendar cal) throws SQLException {
+        ensureNotClosed();
+        ensureHasCurrentRow();
+
+        return getTimestamp(colIdx);
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public Timestamp getTimestamp(String colLb, Calendar cal) throws SQLException {
+        ensureNotClosed();
+        ensureHasCurrentRow();
+
+        throw new UnsupportedOperationException();
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public Timestamp getTimestamp(String colLb) throws SQLException {
+        ensureNotClosed();
+        ensureHasCurrentRow();
+
+        throw new UnsupportedOperationException();
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public InputStream getAsciiStream(int colIdx) throws SQLException {
+        ensureNotClosed();
+
+        throw new SQLFeatureNotSupportedException("Streams are not supported.");
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public InputStream getAsciiStream(String colLb) throws SQLException {
+        ensureNotClosed();
+
+        throw new SQLFeatureNotSupportedException("Streams are not supported.");
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public InputStream getUnicodeStream(int colIdx) throws SQLException {
+        ensureNotClosed();
+
+        throw new SQLFeatureNotSupportedException("Streams are not supported.");
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public InputStream getUnicodeStream(String colLb) throws SQLException {
+        ensureNotClosed();
+
+        throw new SQLFeatureNotSupportedException("Streams are not supported.");
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public InputStream getBinaryStream(int colIdx) throws SQLException {
+        ensureNotClosed();
+
+        throw new SQLFeatureNotSupportedException("Stream are not supported.");
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public InputStream getBinaryStream(String colLb) throws SQLException {
+        ensureNotClosed();
+
+        throw new SQLFeatureNotSupportedException("Streams are not supported.");
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public SQLWarning getWarnings() throws SQLException {
+        ensureNotClosed();
+
+        return null;
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public void clearWarnings() throws SQLException {
+        ensureNotClosed();
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public String getCursorName() throws SQLException {
+        ensureNotClosed();
+
+        return null;
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public ResultSetMetaData getMetaData() throws SQLException {
+        ensureNotClosed();
+
+        return initMetadata();
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public int findColumn(String colLb) throws SQLException {
+        ensureNotClosed();
+
+        Objects.requireNonNull(colLb);
```

Comment:
I think it should be SQLException, because Javadoc state it for case when resultset does not contain a column labeled as parameter